promql/engine.go

```diff
@@ -823,6 +823,17 @@ func (ng *Engine) getTimeRangesForSelector(s *parser.EvalStmt, n *parser.VectorS
 	return start, end
 }
 
+func (ng *Engine) getSubqueryInterval(path []parser.Node) time.Duration {
```

Comment:
```suggestion
func (ng *Engine) getLastSubqueryInterval(path []parser.Node) time.Duration {
```

Reply:
Just to confirm: you only need the inner-most interval because that is technically the required step for the series. (I see that is what is being done)

Reply:
Yep, that's true. I noticed that after the tests started failing and only then did I think about this case.